segments/vcs_branch.sh

```diff
@@ -3,6 +3,8 @@
 # Source lib to get the function get_tmux_pwd
 source "${TMUX_POWERLINE_DIR_LIB}/tmux_adapter.sh"
 
+TMUX_POWERLINE_SEG_VCS_BRANCH_MAX_LEN_DEFAULT="24"
```

Comment:
Good idea to make it configurable.

A few things are missing from this follow the pattern for configuration in this project:
* Have a `_DEFAULT`, like already done here
* in the `generate_segmentrc()` function (missing in this file, needs to be added), add code to generate a default config file with default value.
* In `__process_settings()` (missing here, needs to be added) set the value or use default value.
* In the `rung_segment()`, use not the `_DEFAULT` but the set value.

It sounds way more complicated than it is. Take a look at the file https://github.com/erikw/tmux-powerline/blob/main/segments/battery.sh and ctrl/cmd+f for `TMUX_POWERLINE_SEG_BATTERY_NUM_HEARTS` to see how it's done, then just copy this same way here :)